mne_features/feature_extraction.py

```diff
@@ -219,7 +223,10 @@ def _apply_extractor(extractor, X, ch_names, return_as_df):
     X = extractor.fit_transform(X)
     feature_names = None
     if return_as_df:
-        feature_names = extractor.get_feature_names()
+        if hasattr(extractor, 'get_feature_names_out'):
```

Comment:
Maybe changing the 4 lines with just:
`feature_names = extractor.get_feature_names_out()` 
will increase code coverage ? (if this is the issue why not merging the PR)